interfaces/apparmor/spec.go

```diff
@@ -260,8 +260,64 @@ func chopTree(path string, assumedPrefixDepth int) (left, right []string, err er
 	return left, right, nil
 }
 
+// WritableMimicProfile writes apparmor rules for a writable mimic at the given path.
```

Comment:
This option is quite obscure in terms of behavior, and thus needs comprehensive documentation so the next reader can make sense of it.

Reply:
Indeed, I will document it extensively